src/User32.Desktop/User32+DESKTOP_ACCESS_MASK.cs

```diff
@@ -0,0 +1,268 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Diagnostics;
+
+    /// <content>
+    /// Contains the <see cref="DESKTOP_ACCESS_MASK"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// The <see cref="Kernel32.ACCESS_MASK"/> type is a bitmask that specifies a set of access rights in the access mask of an access control entry.
+        /// /// This is a set of Specific Rights bitmask that defines access rights and security for desktop objects
+        /// </summary>
+        /// <remarks>
+        /// <see cref="Kernel32.ACCESS_MASK"/> is quite well described here: http://blogs.msdn.com/b/openspecification/archive/2010/04/01/about-the-access-mask-structure.aspx
+        /// Desktop security and access rights are quite well described here: https://msdn.microsoft.com/en-us/library/windows/desktop/ms687391(v=vs.85).aspx
+        /// </remarks>
+        public struct DESKTOP_ACCESS_MASK : IComparable, IFormattable, IComparable<DESKTOP_ACCESS_MASK>, IEquatable<DESKTOP_ACCESS_MASK>, IComparable<Kernel32.ACCESS_MASK>, IEquatable<Kernel32.ACCESS_MASK>
```

Comment:
Ohhh 2.x ?| 🎇🎉🎂

Nice to know we have a long plan of maintaining it all ;-)

I would say we could stick to something like what I have done or a better version of it for now so that we don't need a major refactoring. The ideal thing would be some way we could point to new supersets either using inheritance or generics (which I like most in this case). We could have the SpecificRight and GenericRight be a generic restricted to enumerations on a base abstract class (can we have an abstract struct? I guess not, don't remember being possible to inherit structs).